src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java

```diff
@@ -251,6 +321,22 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * import static javax.swing.WindowConstants.*; // Violation, All static imports comes at bottom
+ *
```

Comment:
i think removing 328 empty line and violation comment at 329 will do the job bcoz that violation as you know due to static import line at 327 and if you remember i asked you to keep 327 to make it simple but if that's not working we can remove it